drawings-gods_data-lists.ttl

```diff
@@ -17,26 +17,33 @@
 	knora-base:isRootNode true ;
 	rdfs:comment "Liste de choix pour les devoirs additionnels"@fr ;
 	rdfs:label "Devoirs additionnels (liste)"@fr , "Additional tasks (list)"@en ;
-	knora-base:hasSubListNode <http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList-attachmentTest> ,
+	knora-base:hasSubListNode <http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList--> ,
+	<http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList-attachmentTest> ,
 	<http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList-drawingParent> ,
 	<http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList-other> .
+	
+	<http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList-attachmentTest> a knora-base:ListNode ;
```

Comment:
`<http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList-attachmentTest>`
Wrong name for the node. Should be: 
`<http://rdfh.ch/lists/drawings-gods-2016-list-AdditionalTaskList-->`